::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 26 Nov 2017 17:25:08 +0100
Subject: [PATCH] baremetal: Add Armband Openstack repos to kvm, cmp

Armband provides one APT repository that should be added to all
baremetal nodes explicitly (the other Armband repos are now part of
`linux.system.repo.mcp.openstack` or `linux.system.repo.mcp.extra`):
- linux.system.repo.mcp.armband;

This repo is pre-installed for all infrastructure VMs, so we only
handle it via Salt for baremetal kvm* and cmp* nodes.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/infra/kvm.yml   | 1 +
 .../classes/cluster/baremetal-mcp-pike-common-ha/openstack_compute.yml   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/infra/kvm.yml b/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/infra/kvm.yml
index dae8ad6..6ed97fb 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/infra/kvm.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/infra/kvm.yml
@@ -7,6 +7,7 @@
 ##############################################################################
 ---
 classes:
+  - system.linux.system.repo.mcp.armband
   - system.linux.system.repo.glusterfs
   - service.keepalived.cluster.single
   - system.glusterfs.server.volume.glance
diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/openstack_compute.yml b/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/openstack_compute.yml
index 50241b4..431fde6 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/openstack_compute.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-pike-common-ha/openstack_compute.yml
@@ -7,6 +7,7 @@
 ##############################################################################
 ---
 classes:
+  - system.linux.system.repo.mcp.armband
   - system.linux.system.repo.glusterfs
   - system.linux.storage.loopback
   - system.glusterfs.client.cluster
